toolchain-windows.cmake

```diff
@@ -14,39 +14,25 @@ else()
   set(rocm_bin "C:/hip/bin")
 endif()
 
-#set(CMAKE_CXX_COMPILER "${rocm_bin}/hipcc.bat")
-#set(CMAKE_C_COMPILER "${rocm_bin}/hipcc.bat")
 set(CMAKE_CXX_COMPILER "${rocm_bin}/clang++.exe")
 set(CMAKE_C_COMPILER "${rocm_bin}/clang.exe")
 
-#set(CMAKE_CXX_LINKER "${rocm_bin}/hipcc.bat" )
-
-# TODO remove, just to speed up slow cmake
-set(CMAKE_C_COMPILER_WORKS 1)
-set(CMAKE_CXX_COMPILER_WORKS 1)
-#
+if (NOT python)
+  set(python "python") # take default for windows
```

Comment:
should we not try to take python3?

Reply:
I guess that's a reasonable thing to do, we do require python3 in our README for Windows.